sklearn/utils/testing.py

```diff
@@ -754,10 +765,6 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         _delete_folder(self.temp_folder)
 
 
-with_network = with_setup(check_skip_network)
```

Comment:
We removed those two aliases since they were meant for the CI and they are used nowhere in the codebase.

Reply:
It is also very unlikely that they are used in any downstream projects.